app/frontend/src/lib/currentLocation.js

```diff
@@ -96,8 +96,8 @@ const currentLocation = {
 
 export default currentLocation;
 
-if (navigator.geolocation) {
-  window.addEventListener('DOMContentLoaded', (event) => {
+if (navigator.geolocation && document.querySelector('.js-location-finder')) {
+  window.addEventListener('DOMContentLoaded', () => {
```

Comment:
Should the DOMContentLoaded event listener go outside the if statement?

Reply:
im not sure it would affect it too much, but it is better logic anyway so have updated